scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -68,8 +68,8 @@ task SplitIntervals {
     # Not `volatile: true` since there shouldn't be a need to re-run this if there has already been a successful execution.
   }
 
-  Int disk_size = if (defined(split_intervals_disk_size_override)) then split_intervals_disk_size_override else 10
-  Int disk_memory = if (defined(split_intervals_mem_override)) then split_intervals_mem_override else 16
+  Int disk_size = if (defined(split_intervals_disk_size_override)) then select_first([split_intervals_disk_size_override]) else 10
```

Comment:
miniwdl pointed out this should be addressed.

Reply:
Good catch by miniwdl, I should get in the habit of using that